.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Feature request
+about: Suggest an idea for Trino Python client
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+### Describe the feature
+A clear and concise description of what you want to happen.
+
+### Describe alternatives you've considered
+A clear and concise description of any alternative solutions or features you've considered.
+
+### Additional context
+Please include any other relevant context here.
+
+### Who will this benefit?
+What kind of use case will this feature be useful for? Please be specific and provide examples, this will help us prioritize properly.
+
+### Are you interested in contributing this feature?
```

Comment:
Maybe to make the template a bit shorter we should just ask users to assign themselves to the issue. All Trino contributors that signed the CLA are added to the `contributors` group and should be able to do that. If not, just leave a comment.

Reply:
Yes, just ask people to assign themselves if they're already working on a PR - helps other people who see the issue know to not duplicate work as well.

Reply:
I think why a lot of other open source projects explicitly ask for this is that it increases the influx of new contributors and shows that the community is there to help. This is probably not so useful for a long time Trino contributor. See eg https://github.com/apache/airflow/issues/new?assignees=&labels=kind%3Abug%2Carea%3Acore&template=airflow_bug_report.yml

Reply:
Good point. Let's keep this and we can assign (or tell them to assign themselves). Let's iterate. Doesn't need to be perfect on first go.